doc/notebooks/jupyter_tutorial.ipynb

```diff
@@ -234,6 +235,40 @@
     "raleigh_map.add_vector(\"roadsmajor\")"
    ]
   },
+  {
```

Comment:
Couple points here: 
1. This works only with ipyleaflet, that needs to be mentioned there.
2. I would put it either at the beginning of `Interactive Map Display` section or at the end of it.
3. Please shorten it, one or 2 sentences each, I think it's not that hard to use (hopefully) so shorter description should be sufficient. But you can put the longer version in the final report.

Reply:
Yes, no worries.